src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,165 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.Scanner;
+
+/**
+ * A to-do bot.
+ *
+ * @author Erwin Quek
+ * @version CS2103 AY21/22 Sem 1
+ */
+public class Duke {
+
+    protected static Storage storage;
+    protected static TaskList tasks;
+    protected static Ui ui;
+
+    /**
+     * Constructor to create Duke.
+     * @param filePath Takes in a filepath
+     * @throws FileNotFoundException Throws an error when file not found
+     */
+    public Duke(String filePath) throws FileNotFoundException {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        tasks = new TaskList(storage.load());
+    }
+
+    /**
+     * Method to run the bot.
+     */
+    public void run() {
+        ui.showWelcomeMessage();
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                ui.readCommand();
+                Scanner s = new Scanner(System.in);
+                String command = s.nextLine();
+                isExit = Parser.parse(command);
+                //Command c = duke.Parser.parse(fullCommand);
+                //c.execute(tasks, ui, storage);
+            //} catch (duke.DukeException e) {
+            //    ui.showError(e.getMessage());
```

Comment:
Maybe you should remove irrelevant comments?

Reply:
I was thinking if i want to work on this later on. but I think I will remove it! Thank you